c7n/resources/appelb.py

```diff
@@ -713,6 +713,8 @@ def process_alb(self, client, alb):
                 client.delete_load_balancer, LoadBalancerArn=alb['LoadBalancerArn'])
         except client.exceptions.LoadBalancerNotFoundException:
             pass
+        except client.exceptions.ResourceInUseException:
+            pass
         except client.exceptions.OperationNotPermittedException as e:
             self.log.warning(
                 "Exception trying to delete ALB: %s error: %s",
```

Comment:
We silently skip "not found" exceptions because there's nothing to be done. Handling "resource in use" feels more closely related to "operation not permitted" though, where we likely want to drop a note about why we're skipping.

(Also it seems like a good time to not overly commit to "ALB" in the message given that we could be dealing with an NLB instead.)

```suggestion
        except (
            client.exceptions.OperationNotPermittedException,
            client.exceptions.ResourceInUseException
        ) as e:
            self.log.warning(
                "Exception trying to delete load balancer: %s error: %s",
```